SocialFilter/sections/specific.txt

```diff
@@ -46,6 +46,7 @@ archives.gov,sportnews.to,nationalchronicle.in,analogictips.com,linearmotiontips
 ! ##.sns_area
 sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 !
+wpolityce.pl##.article__social-media
```

Comment:
You also remove these buttons.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/145566154-3bc77b0f-02b5-47bf-9623-89f00fc663c3.png)
</details><br/>
To avoid this, please replace the rule to the rules below

```
wpolityce.pl##.article-sm__item--title
wpolityce.pl##.article-sm__item--title ~ li > *:not(.js-comments):not(.js-print-publication-button)
```